Sources/PromotedAIMetricsSDK/NetworkConnection.swift

```diff
@@ -34,12 +33,25 @@ public protocol NetworkConnection {
   ///   any errors that occurred in protobuf serialization *prior to*
   ///   the network operation. Errors resulting from the network operation
   ///   are passed through `callback`.
-  func sendMessage(_ message: Message, url: URL, clientConfig: ClientConfig,
+  func sendMessage(_ message: Message, clientConfig: ClientConfig,
                    callback: Callback?) throws
 }
 
 // MARK: -
 extension NetworkConnection {
+  func metricsLoggingURL(clientConfig: ClientConfig) throws -> URL {
+    #if DEBUG
```

Comment:
Is it a common practice for local workstations and test/dev environments to use DEBUG?

Reply:
There's no single convention about local build vs. prod build. I considered a few things:

1. Checking if we're running in the iOS simulator (this misses if we're running locally on device).
2. Checking if we're running on an App Store build (this is somewhat complex and probably more trouble than it's worth).
3. Checking if we're running in debug.

Given that, when people are running locally, it's almost always for debug/development, choice 3 seemed safest.